gravitee-am-gateway/gravitee-am-gateway-handler/gravitee-am-gateway-handler-account/src/main/java/io/gravitee/am/gateway/handler/account/resources/AccountEndpointHandler.java

```diff
@@ -153,6 +184,11 @@ private DefaultUser convertUserToPrincipal(RoutingContext routingContext, User u
     }
 
     public void redirectForgotPassword(RoutingContext routingContext) {
+        if (tokenCanHandleResetPassword(routingContext)){
```

Comment:
This error message for get request seems slightly misleading. The exception is thrown when the token can handle the request. Did you miss `!` in the if condition?